admin/templates/configuration/etc/log4cxx.properties

```diff
@@ -5,6 +5,6 @@
 log4j.rootLogger=DEBUG, QLogger
 log4j.appender.QLogger=org.apache.log4j.ConsoleAppender
 log4j.appender.QLogger.layout=org.apache.log4j.PatternLayout
-log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [%t] %-5p %c{2} (%F:%L) - %m%n
+log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [PID:%X{PID}, %t] %-5p %c{2} (%F:%L) - %m%n
```

Comment:
I do not see anybody "agreeing" that qserv have to log PID. For PID is only useful in situations when multiple processes log to the same file simultaneously, PID would help to disentangle interleaved messages then. In all other cases its only taking space on a line.
That log4cxx file is not qserv-specific, I think it can be used by default by other apps. What happens when you use %X{PID} but there is no PID in MDC?